ci/travis.sh

```diff
@@ -92,7 +92,7 @@ else
     mkdir empty
     cd empty
 
-    pytest -W error -ra -v --cov=trio_asyncio --cov-config=../.coveragerc --verbose ../tests
+    pytest -ra -v --cov=trio_asyncio --cov-config=../.coveragerc --verbose ../tests
```

Comment:
Is this change intentional?

Reply:
Yes -- I created a pytest.ini with a more detailed warnings filter. (The loop= argument is deprecated in several asyncio methods on 3.8+. I didn't think it advisable to remove support for passing the loop just yet.)